src/System.Drawing.Common/src/System/Drawing/FontFamily.cs

```diff
@@ -236,14 +160,8 @@ private void Dispose(bool disposing)
                     Debug.Assert(status == SafeNativeMethods.Gdip.Ok, "GDI+ returned an error status: " + status.ToString(CultureInfo.InvariantCulture));
 #endif
                 }
-                catch (Exception ex)
+                catch (Exception ex) when (!ClientUtils.IsCriticalException(ex))
```

Comment:
We've changed this pattern before after discussing with @stephentoub in a previous PR. I think we reached the conclusion it has no functional difference as an exception is thrown if it's critical, but caught If it is not